boards/riscv/it8xxx2_evb/doc/index.rst

```diff
@@ -0,0 +1,72 @@
+.. _it8xxx2_evb:
+
+ITE IT8XXX2 series
+######################
+
+Overview
+********
+
+The IT8XXX2 is a 32-bit RISC-V Micro-controller and a
+highly integrated embedded controller with system functions suitable for mobile system applications.
+To find out more about ITE, visit our World Wide Web at:
+`ITE's website <http://www.ite.com.tw>`_.
+
+Hardware
+********
+The IT8XXX2 series do not support exactly the same hardware features.
```

Comment:
What is this statement trying to say?  Confused by the 'do not support..'

Reply:
Sorry, It's means that the it8xxxx series have multiple type of mcu, they have different hardware features
for example,
the it81302 is 60 kb sram
the it83202 is 256kb sram
We will rewrite the statement to avoid confusig.